purefa/README.md

```diff
@@ -0,0 +1,123 @@
+# Agent Check: PureFA
+
+## Overview
+
+This check monitors the [Pure Storage FlashArray][1] through the [Datadog Agent][2] and the [Pure Storage Prometheus exporter][1].
+
+Note: This integration requires OpenMetricsBaseCheckV2 which is available in Agent v7.26.x+ and requires Python 3
```

Comment:
```suggestion
Note: This integration requires:
-  OpenMetricsBaseCheckV2, which is available in Agent v7.26.x+ 
- Python 3
- Use of the [Pure Storage Prometheus exporter][1]
```

Does the user have to download/do any installation for the Prometheus exporter? If so, that should be mentioned here as well.

Reply:
Ack